src/chrome-launcher.ts

```diff
@@ -90,6 +85,19 @@ async function launch(opts: Options = {}): Promise<LaunchedChrome> {
   return {pid: instance.pid!, port: instance.port!, kill, process: instance.chrome!};
 }
 
+async function killAll(): Promise<boolean> {
```

Comment:
a bit odd IMO to return a boolean here since it's not really actionable to the user, what about returning an array of the errors so they could at least log the messages if they care about them?

Reply:
yeah I didn't know if you'd balk at adding complexity so I hedged with a simple errors occurred flag.

I'm not sure the array is the best idea, but it might be. I agree providing more information, if we are going to provide some, does make sense. 

I'll add something.

thanks for the review 😊